about.html

```diff
@@ -28,23 +28,29 @@
         <div class="navbar">
           <img src="./images/icons/Union.svg" alt="Union img" id="union" />
           <div class="container">
-            <a href="/index.html"
-              ><img
-                src="./images/icons/homeicon.jpeg"
-                alt="home icon"
-                id="homeicon"
+            <div class="containercontent">
+            <a href="./index.html"
+              ><img src="./images/icons/homeicon.jpeg" alt="home icon" id="homeicon"
             /></a>
-            <ul>
-              <li><a href="./about.html">About</a></li>
-              <li><a href="#">Contact</a></li>
-              <li><a href="#">Assets</a></li>
-              <li><a href="#">News</a></li>
-              <li><a href="#" id="chart">Chart</a></li>
-            </ul>
+             <h4>BCB-B2C</h4>
+            </div>
+            <!-- navbar sign up -->
+             <div class="navbarcontent">
+              <ul>
+                <li><a href="https://mail.google.com/mail/u/0/">Contact</a></li>
+                <li><a href="https://www.investopedia.com/terms/b/business-asset.asp">Assets</a></li>
+                <li><a href="https://www.bbc.com/news/business">News</a></li>
+                <li><a href="https://www.smartdraw.com/organizational-chart/" id="chart">Chart</a></li>
+              </ul>
+              <a href="https://www.behance.net/gallery/29845175/CC-Global-Summit-2015" id="signup">Sign Up</a>
+             </div>
+  
+            
           </div>
         </div>
         <!-- DROPDOWN -->
         <section class="dropdown">
+          <h2></h2>
```

Comment:
- [ ] Please add the `./index.html` to the `href` on line 58 so that when you click on home from the mobile menu it takes you to the home page and does not throw an error🤔😉